core_backend/app/configs/llm_prompts.py

```diff
@@ -113,12 +113,32 @@ def get_supported_languages(cls) -> list[str]:
 # ---- Paraphrase question
 PARAPHRASE_FAILED_MESSAGE = "ERROR: CAN'T PARAPHRASE"
 PARAPHRASE_INPUT = f"""
-    You are a high-performing paraphrase bot.
-    You support a question-answering service.
-    The user has asked a question in English, paraphrase it to focus on
-    the question. Be succinct and do not include any unnecessary information.
-    Ignore any redacted and offensive words.
-    If no paraphrase is possible, respond with {PARAPHRASE_FAILED_MESSAGE}
+    You are a high-performing paraphrasing bot. You support a question-answering
+    service. The user has asked a question in English. Do not answer the question,
```

Comment:
Paraphrase often fails when the user hasn't actually asked a question but says "data systems" as a search term - what do we want to do in such cases?


If we want to stick with the current flow, we can add the clarification in the prompt that "the user has sent a message, if it's not a question, return `PARAPHRASE_FAILED_MESSAGE`, but if it's a questions, praphrase etc...

Otherwise, we can have another guardrail that just checks if it's actually a question or not and errors out if not, but I'm a bit hesitant about adding more guardrails + latency etc.

Reply:
Yep - The new prompt handles that. Check out the test cases! If it's a question, it paraphrases. if it's not, it just returns the same statement.